tests/test_test_schema_section.py

```diff
@@ -78,21 +78,19 @@ def test_wrong_type():
 
 
 def test_min_length_violated():
-    # TODO: Make this pass (not implemented yet)
     """ Not adhering to minlength limitations should raise an error """
     with pytest.raises(
         DocumentationError,
-        match="Mismatched content. Response string violates the minimum string " "length defined in the schema",
+        match="The length of aa exceeds the minimum allowed length of 3",
     ):
         tester.test_schema_section(example_schema_string, "a" * 2)
 
 
 def test_max_length_violated():
-    # TODO: Make this pass (not implemented yet)
     """ Not adhering to maxlength limitations should raise an error """
     with pytest.raises(
         DocumentationError,
-        match="Mismatched content. Response string violates the maximum string " "length defined in the schema",
+        match="The length of aaaaaa exceeds the maximum allowed length of 5",
```

Comment:
You are testing specific messages. This is an antipattern. 

I suggest moving the error messages into constants and use .format:

MY_MSG = "message {0}"

And then use the constant in both the code and test.

Reply:
Yep, agree. Think I will do that in a separate PR following this one.